src/pages/Thanks/ThanksPage.vue

```diff
@@ -116,13 +150,13 @@ export default {
 			// The default empty object and the v-if will prevent the
 			// receipt from rendering in the rare cases this query fails.
 			// But it will not throw a server error.
-			this.receipt = _get(data, 'shop.receipt');
-			const loansResponse = _get(this.receipt, 'items.values', []);
+			this.receipt = data?.shop?.receipt;
+			const loansResponse = this.receipt?.items?.values || [];
```

Comment:
I've been using the "nullish" `??` in these cases.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_Coalescing_Operator

Reply:
Def would like to hear your thoughts on this operator there's likely other locations we'll want to update.